tests/test_utils.sh

```diff
@@ -847,6 +847,16 @@ function test_grep_fields {
     rm -f "$filename"
 }
 
+function test_substitute {
+    local sub=(
+        "When=NOK"
+        "What?=NOPE"
+        "What=OK"
+    )
+    _lp_substitute "What" ${sub[@]}
```

Comment:
Pretty sure this should be quoted
```suggestion
    _lp_substitute "What" "${sub[@]}"
```